R/verb-copy-to.R

```diff
@@ -227,11 +227,25 @@ sql_values_clause <- function(con, df, row = FALSE, lvl = 0) {
   rows <- rlang::exec(paste, !!!escaped_values, sep = ", ")
   rows_sql <- sql(paste0(if (row) "ROW", "(", rows, ")"))
 
-  union_query <- set_op_query(
-    null_row_query,
-    sql_format_clauses(list(sql_clause("VALUES", rows_sql)), lvl = lvl + 1, con = con),
-    type = "UNION ALL",
-  )
+  rows_clauses <- list(sql_clause("VALUES", rows_sql))
+
+  if (derived) {
+    rows_query <- sql_format_clauses(rows_clauses, lvl = 0, con = con)
```

Comment:
The formatting is incorrect when there are many values

```sql
SELECT CAST("x" AS INTEGER) AS "x", CAST("y" AS TEXT) AS "y"
FROM (
  (
    SELECT NULL AS "x", NULL AS "y"
    WHERE (0 = 1)
  )
  UNION ALL
  (
    SELECT *
    FROM (
VALUES
  (1, 'a'),
  (2, 'b'),
  (3, 'c'),
  (4, 'd'),
  (5, 'e'),
  (6, 'f'),
  (7, 'g'),
  (8, 'h')
    ) drvd("x", "y")
  )
) "values_table"
```

but I don't think it's worth spending much time on fixing this.
Maybe add a comment that this is a known issue?